bundles/io/org.eclipse.smarthome.io.rest.core/src/main/java/org/eclipse/smarthome/io/rest/core/persistence/PersistenceResource.java

```diff
@@ -58,18 +68,22 @@
  * store
  *
  * @author Chris Jackson - Initial Contribution
+ * @author Chris Jackson - Updated to add support for ModifiablePersistenceService
```

Comment:
Don't add a second line with the same author - rather combine the description text (although initial contrib is still good enough as long as you are the only author).